registries/registries.py

```diff
@@ -38,7 +38,21 @@ def write_file(filename, data):
         f.write(data)
 
 
+def trim_suffix(s, suffix):
```

Comment:
Just use rstrip.
```
 python
Python 2.7.14 (default, Feb 27 2018, 20:43:24) 
[GCC 7.3.1 20180130 (Red Hat 7.3.1-2)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> s="This is a test///"
>>> s.rstrip("/")
'This is a test'
```

Reply:
`rstrip` would remove any number of trailing slashes, but we opted to remove at most one (see https://github.com/vrothberg/image/blob/1c2d9a32e63ab66cb512803622a9e0c414e89381/pkg/sysregistries/system_registries_test.go#L42). Should we adjust that in `containers/image` as well?

Reply:
I see no reason why you would ever have a url of a registry that ends with a slash, so removing only one versus all seems like the wrong solution.

Reply:
I updated this PR to use `str.rstrip()`. Shall I update `containers/image` to strip all as well?

Reply:
Yes I think so.

Reply:
https://github.com/containers/image/pull/430